sphinx_gherkindoc/parsers/behave.py

```diff
@@ -41,7 +41,9 @@ def __init__(self, root_path: str, source_path: str):
     @property
     def scenarios(self) -> List[Scenario]:
         """Wrap Behave Scenarios to include description processing."""
-        return [Scenario(s) for s in self._data.scenarios]
+        if hasattr(self._data, "scenarios"):
+            return [Scenario(s) for s in self._data.scenarios]
+        return []
```

Comment:
Thanks!
I'd like to see this a bit simpler though, `getattr` has a default value, so I think the change could be to keep the original `return` and tweak `self._data.scenarios` -> `getattr(self._data, "scenarios", [])`

Reply:
Yes, that is much cleaner. I tried using `None` in the getattr at first and didn't think father than that.